tox.ini

```diff
@@ -2,13 +2,23 @@
 envlist = py26,py27,py33,py34,py35,pypy
 
 [testenv]
+deps =
+    nose
+    mock
+
+changedir={toxinidir}/tests
```

Comment:
I like spaces around `=`, but if you change the commands to run nosetests, changedir won't be needed at all.